lambda/format-user-services/tests/format-user-services.test.ts

```diff
@@ -18,7 +18,7 @@ import {
   makeServiceRecord,
   makeSQSInputFixture,
 } from "./testHelpers";
-import { UserServices, Service } from "../models";
+import { UserServices, Service, TxmaEvent } from "../models";
```

Comment:
looks like there's a linting error here.
Let's have a chat about getting prettier set up in vscode it should autoremove things like this under some conditions!

Reply:
ah wait, I see there's a clash with types declared below 🤔 
Ok perhaps best that this line becomes:

```suggestion
import type { UserServices, Service, TxmaEvent } from "../models";
```

Then remove TxmaEvent if it's not being used
and lines 144 and 161 should be `txmaEvent` to follow converntion as constants I think!